### PR TITLE
Add 404 page in webbsite

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
Add 404 page in webbsite

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [ ] `apps/stats-dapp`
- [x] `apps/webbsite`
- [ ] `apps/faucet`
- [ ] `apps/tangle-website`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
#1252 

### Screen Recording
_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/42c2fe64-c643-4a30-bd2a-090b111c9a29

-----
### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: apps/webbsite/src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            variant="mkt-h1"
+            className="text-center text-mono-200 font-black !text-[86px] !leading-[72px]"
+          >
+            404
+          </Typography>
+          <Typography
+            variant="mkt-subheading"
+            className="text-center text-mono-140 font-medium"
+          >
+            It looks like this page doesn&apos;t exist.
```
Comment (shekohex) [inline apps/webbsite/src/pages/404.tsx:18]: Some cool messages that we can show randomly for the user:
1. Oops! It seems the page you were trying to access has been swept away by a cosmic privacy nebula, ensuring that your online journey remains hidden from prying eyes. We're navigating the celestial depths to retrieve it for you.

2. Attention, galactic explorers! The privacy warp drive of our Webb application has temporarily redirected the page you were seeking to a hidden corner of the universe, shielding your personal information from the digital cosmos. We're charting a course to bring it back to your orbit.

3. Privacy breach intercepted! The Webb application's advanced encryption system has detected a rogue data comet attempting to access the page you were looking for. Rest assured, our interstellar cyber guardians are working diligently to thwart this intrusion and restore your secure browsing experience.

4. Warp speed encryption engaged! The page you were attempting to reach has temporarily shifted to a parallel privacy dimension, where its sensitive content is safeguarded against the prying algorithms of the digital galaxy. We're synchronizing our cryptographic algorithms to retrieve it without compromising your privacy.

5. Transmitting from a secret asteroid base! The page you're searching for is undergoing a stealthy privacy upgrade within our remote celestial sanctuary. We're reinforcing its security protocols to keep your personal information shielded from the cosmic data scavengers.

6. A quantum leap for privacy! The Webb application's cutting-edge privacy technology has initiated a subatomic encryption process, rendering the page you sought temporarily indistinguishable from the vast expanse of the cosmos. We're harnessing the power of quantum entanglement to bring it back to your digital universe.

7. Lost in the astral plane of privacy! The page you were aiming for has embarked on an otherworldly journey through a parallel dimension, where it remains hidden from prying eyes. We're consulting with extraterrestrial privacy experts to bring it back to our realm.

8. Cloaked in the celestial veil of privacy! The page you're trying to access has slipped into the protective shadow of a distant black hole, where it remains invisible to unauthorized entities. We're calculating a precise trajectory to retrieve it without disturbing the delicate balance of the privacy cosmos.

9. Beyond the event horizon of privacy! The page you seek has ventured into the depths of a privacy singularity, where its data is shielded by the immense gravitational forces of encryption. We're deploying advanced privacy probes to navigate this cosmic anomaly and restore your access.

10. Engaging stealth protocol from a lunar outpost! The page you're attempting to find has taken refuge on the far side of the moon, shielded by the lunar surface's natural privacy defenses. We're coordinating with our lunar privacy squadron to reestablish the connection and ensure your online privacy remains intact.

Comment (drewstone) [reply apps/webbsite/src/pages/404.tsx:18]: Yea let's keep it simple lol

Comment (monaiuu) [reply apps/webbsite/src/pages/404.tsx:18]: lol what about this:

Screenshot 2023-05-25 at 2 42 41 PM

Comment (monaiuu) [reply apps/webbsite/src/pages/404.tsx:18]: Let's also redirect users to report bug or get support via discord with the [Get Help] button. Maybe that's more helpful for users.

Comment (monaiuu) [reply apps/webbsite/src/pages/404.tsx:18]: cc @vutuanlinh2k2 @drewstone @shekohex

Comment (shekohex) [reply apps/webbsite/src/pages/404.tsx:18]: Good idea!

Comment (vutuanlinh2k2) [reply apps/webbsite/src/pages/404.tsx:18]: @monaiuu If we add the [Get Help] button, where should we put that? I think if we either add the button on the same line or below the 2 existing buttons, the UI will not be good-looking. Maybe you can help me update on the Figma file if we implement it 😊

Comment (monaiuu) [reply apps/webbsite/src/pages/404.tsx:18]: sorry just saw this, I think we should replace replace the [Explore the Blog] to [Get Help], redirecting users to Webb's Discrod Feedback-Suggestion Channel (https://discord.com/channels/833784453251596298/845031666987696158)

**Pull request merged**